keras_rcnn/models.py

```diff
@@ -49,15 +48,17 @@ class ResNet50RCNN(RCNN):
     :param inputs: input tensor (e.g. an instance of `keras.layers.Input`)
     :param classes: integer, number of classes
     :param rois: integer, number of regions of interest per image
+    :param blocks: list of blocks to use in ResNet50
 
     :return model: a functional model API for R-CNN.
     """
 
-    def __init__(self, inputs, classes, rois=300):
+    def __init__(self, inputs, classes, rois=300, blocks=[3, 4, 6]):
         # ResNet50 as encoder
-        encoder = keras_resnet.models.ResNet50
+        image, _, _ = inputs
+        features = keras_resnet.models.ResNet50(image, blocks=blocks, include_top=False).output
```

Comment:
I was thinking about this one ... do we want to call the model as a function (ie. `keras_resnet.models.ResNet50(blocks=blocks, include_top=False)(image)`) or do we want to incorporate every layer in the RCNN model? The difference is that in the summary, you'd see all the layers (the way it is now) or you would see a 'layer' called ResNet50. Let me know what you think.